### ISSUE TITLE
[Bug] Fenix App crashes when performing a searchon indiamart.com/postbuy.php

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [mozilla-mobile/fenix#6326]
## Steps to reproduce
1. Navigate to https://m.indiamart.com/postbuy.php 
2. Perform a search after a product (e.g toy).
3. Observe  behavior.

### Expected behavior
Search results are displayed.

### Actual behavior
Application crashes.

### Notes
1. Not reproducible on Firefox Nightly 71.0a1 (2019-10-26) and Chrome 77.0.3865.90
2. Reproducible: 100%
3. Gif attached.
4. The issue is reproducible according to the steps to reproduce, the first time. After that just navigating to the page, the app crashes.

### Device information

* Android device: Huawei P20 Lite (Android 8.0.0) - 1080 x 2280 pixels, 19:9 ratio (~432 ppi density), Samsung Galaxy S7 Edge (Android 8.0.0) - Resolution 1440 x 2560 pixels (~534 ppi pixel density)
* Fenix version: Firefox Preview Nightly 191029 (🦎: 72.0a1-20191025095546)

![CrashSearch](https://user-images.githubusercontent.com/12184325/67772891-874a2380-fa63-11e9-8c69-e57ac756b96c.gif)

**Comment by mcarare** [comment]
Note: The crash does not occur with build variant `geckoBetaDebug`, only with `geckoNightlyDebug`.

**Comment by ekager** [comment]
Filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1593156

**Comment by perryjiang** [comment]
On Bugzilla this was marked as a duplicate of https://bugzilla.mozilla.org/show_bug.cgi?id=1409979, which should be resolved soon once it lands in mozilla-central.

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **Nightly Build #20310607 from 1/31** using the following devices:

• Google Pixel 3a (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

► Video
![20200131_115748](https://user-images.githubusercontent.com/51314259/73530772-c5b7fb80-4421-11ea-99f7-f99cdbcd701f.gif)
